os/arch/Kconfig

```diff
@@ -21,6 +21,7 @@ config ARCH_XTENSA
 	bool "Xtensa"
 	select ARCH_HAVE_STACKCHECK
 	select ARCH_HAVE_CUSTOMOPT
+	select ARCH_VECNOTIRQ
 	---help---
 		Cadence Tensilica Xtensa architectures.
 endchoice
```

Comment:
why omit esp32_DevKitC/hello_with_tash to add CONFIG_ARCH_VECNOTIRQ
why not add it on esp32_DevKitC board.

Reply:
There are 7 defconfigs for esp32 board. Why do you add it on only 3?

Reply:
Others defconfig files are already added.  
The new submitted is not included by the submitter.